packages/core-commands/src/admin-navigation-commands.js

```diff
@@ -77,6 +77,12 @@ const getAddNewPageCommand = () =>
 					label: __( 'Add new page' ),
 					icon: plus,
 					callback: addNewPage,
+					keywords: [
+						__( 'page' ),
+						__( 'new' ),
```

Comment:
I expect there are languages in which "new" might not feature in the translation for `"Add new page"` so it seems safe to duplicate, at least in English.